js/modules/inputValidation.js

```diff
@@ -9,93 +8,92 @@ const REGEXP = {
 const RULES = [
 	{
 		name: "name",
-		validate: function(input){
+		validate: function (input) {
 			return REGEXP.NAME.test(input.value);
 		}
 	},
 	{
 		name: "phone",
-		validate: function(input){
+		validate: function (input) {
 			return REGEXP.PHONE.test(input.value);
 		}
 	},
 	{
 		name: "checkbox",
-		validate: function(input){
+		validate: function (input) {
 			return input.checked;
 		}
 	}
 ]
 
 forms.forEach((form) => form.addEventListener("submit", handleFormSubmit));
 
-function validateField(input){
+function validateField(input) {
 	const checkings = [];
 
 	RULES.forEach((rule) => {
-		if(Object.hasOwn(input.dataset, rule.name)){
+		if (Object.hasOwn(input.dataset, rule.name)) {
 			const isValid = rule.validate(input);
 			checkings.push(isValid);
 		}
 	});
 	return checkings.every((item) => item);
 }
 
-function isFormValid(form){
+function isFormValid(form) {
 	const checkings = [];
-	const filteredFields = Array.from(form.elements).filter((element)=>{
+	const filteredFields = Array.from(form.elements).filter((element) => {
 		return Object.hasOwn(element.dataset, "toValidate");
 	});
 
-	filteredFields.forEach((element)=> {
-
+	filteredFields.forEach((element) => {
 		const isValid = validateField(element);
```

Comment:
use word "field" or "input" instead of element here and below in this function as well, so that it could be simpler to read the code

Reply:
Done.